Source/JavaWrappers.cpp

```diff
@@ -274,6 +282,112 @@ namespace java::lang
     }
 }
 
+namespace java::websocket
+{
+    jclass g_webSocketClass{};
+    void InitializeJavaWebSocketClass(jclass webSocketClass, JNIEnv* env) {
+        g_webSocketClass = (jclass) env->NewGlobalRef(webSocketClass);
+    }
+    void DestructJavaWebSocketClass(JNIEnv* env) {
+        env->DeleteGlobalRef(g_webSocketClass);
+    }
```

Comment:
I was thinking passing the class to the constructor, but I guess this works also and is perhaps easier to deal with if we can find a better way to handle it. I would recommend making these static methods of the WebSocketClient class instead. This class is specific to WebSocketClient right?